kafka-ui-api/src/main/java/com/provectus/kafka/ui/service/rbac/extractor/GithubAuthorityExtractor.java

```diff
@@ -83,7 +83,9 @@ public Mono<Set<String>> extract(AccessControlService acs, Object value, Map<Str
 
     final Mono<List<Map<String, Object>>> userOrganizations = webClient
         .get()
-        .uri("/orgs")
+        .uri(uriBuilder -> uriBuilder.path("/orgs")
+            .queryParam("per_page", 100)
```

Comment:
please add comment that it is a max value for github api / make a constant with descriptive name